web/app/js/components/util/TapUtils.jsx

```diff
@@ -343,9 +342,7 @@ export const srcDstColumn = (d, resourceType, ResourceLink) => {
   };
 
   let baseContent = (
-    <Link to="#" onClick={linkFn}>
-      <OpenInNewIcon fontSize="small" />
-    </Link>
+    <OpenInNewIcon fontSize="small" style={{color: "var(--linkblue)"}} onClick={linkFn} />
```

Comment:
To confirm, you're hardcoding the style here in order not to use the `Link` component? Could we use `withStyles` instead?

Reply:
Yeah, this was a pretty big hack. The previous code didn't lint because it didn't pass accessibility checks. We shouldn't have been using an empty link to trigger an onclick event. We could definitely use `withStyles`, and that's how I'd fix it in a component, but this code appears in a function that returns raw JSX. We should instead convert the whole function to a component, but that felt like a bigger change. If it's ok by you I'm inclined to leave this as is.

Reply:
Thanks for the explanation - fine by me.